Pod/Classes/Core/public/WistiaAPI.swift

```diff
@@ -36,6 +36,8 @@ public class WistiaAPI {
     fileprivate let apiToken: String?
     public let sessionManager: SessionManager
 
+    public static let anonymous = WistiaAPI(withoutApiToken:true)
```

Comment:
This needs documentation if it's going to be public.  Although it could probably just be `internal` for now.